conversations.yaml

```diff
@@ -301,16 +329,21 @@ components:
           type: string
         profile_pic:
           type: string
-          description: a publicly accessible image URL.
+          format: uri
+          description: 
+            A publicly-accessible URL to the customer's avatar or 
+            profile picture.
         locale:
           type: string
-          description: the ISO-639 or ISO-3166 country code.
+          description: The ISO-639 or ISO-3166 country code.
         timezone:
           type: integer
+          format: int64
           description: "Time zone offset, the number relative to GMT, (ex: -8)."
         phone_number:
           type: string
-          description: must be an E.164 format number.
+          pattern: ^\+[1-9]\d{1,14}$
```

Comment:
https://stackoverflow.com/questions/6478875/regular-expression-matching-e-164-formatted-phone-numbers

According to that link, the pattern should be `^\+?[1-9]\d{1,14}$`.  Can you comment on whether the `+` is optional for Medallia?